client/src/js/hmm/components/__tests__/InstallOption.test.js

```diff
@@ -0,0 +1,62 @@
+jest.mock("../../../utils/utils");
+import { InstallOption, mapStateToProps, mapDispatchToProps } from "../InstallOption";
+import { checkAdminOrPermission } from "../../../utils/utils";
+
+describe("InstallOption", () => {
+    let props;
+    beforeEach(() => {
+        props = {
+            canInstall: true,
+            releaseId: "foo"
+        };
+    });
+
+    it("should render when [canInstall=true]", () => {
+        const wrapper = shallow(<InstallOption {...props} />);
+        expect(wrapper).toMatchSnapshot();
+    });
+    it("should render when [canInstall=false]", () => {
+        props.canInstall = false;
+
+        const wrapper = shallow(<InstallOption {...props} />);
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("mapStateToProps", () => {
+    const state = {
+        hmms: {
+            status: {
+                release: {
+                    id: "foo"
+                }
+            }
+        },
+        account: {
+            administrator: false,
+            permissions: { modify_hmm: true }
+        }
+    };
+    it("should return mapStateToProps in props", () => {
+        checkAdminOrPermission.mockReturnValue(true);
+        const result = mapStateToProps(state);
+        expect(result).toEqual({
+            canInstall: true,
+            releaseId: "foo"
+        });
+    });
+    it("checkAdminOrPermission should return state.account.permissions[permission]", () => {
+        expect(checkAdminOrPermission).toHaveBeenCalledWith(state, "modify_hmm");
+    });
+});
+
+describe("mapDispatchToProps", () => {
+    const dispatch = jest.fn();
```

Comment:
This `describe()` block should have a nested `it()` with name 'should return onInstall() in props`.